timesketch/lib/analyzers/sigma_tagger.py

```diff
@@ -175,19 +174,19 @@ def run(self):
         return output_string
 
 
-class LinuxRulesSigmaPlugin(SigmaPlugin):
+class SampleRulesSigmaPlugin(SigmaPlugin):
     """Sigma plugin to run Linux rules."""
 
-    _RULES_PATH = '../../../data/sigma/rules/linux'
+    _RULES_PATH = '../../../data/sigma_sample'
 
-    NAME = 'sigma_linux'
+    NAME = 'sigma_sample'
 
-class WindowsRulesSigmaPlugin(SigmaPlugin):
+class AllRulesSigmaPlugin(SigmaPlugin):
```

Comment:
Do we intend to have all Sigma rules under this analyzer? If that is the case, can we call it simply `SigmaPlugin`?